ksql-engine/src/test/resources/query-validation-tests/insert-into.json

```diff
@@ -0,0 +1,30 @@
+{
+  "comments": [
+    "You can specify multiple statements per test case, i.e., to set up the various streams needed",
```

Comment:
super nit: having this generic comment copied everywhere isn't of any value. I think we should either delete the comment or put a more useful / targeted comment here.  In this instance, probably just remove it.